spec.md

```diff
@@ -2692,6 +2692,15 @@ module MyModule :
    out <= r$0
 ```
 
+# Semantics of Values
+
+FIRRTL is defined for 2-state logic.  The behavior of a generated circuit in a
```

Comment:
This should probably be `2-state (boolean) logic` or something like that. We need to define what these two states are. If you want to rely on a common understanding around Verilog, we would have to cite the Verilog spec.